debitFeature

```diff
@@ -0,0 +1,3 @@
+This is debit card
+adding this feature
```

Comment:
WTF.. Can you please add another line after 2 and line.

Reply:
resolved